2.8.0/todo/result_onnx.md

```diff
@@ -25,60 +25,144 @@ The categories below are as follows:
 
 ## onnx
 ### bc breaking
-- [ONNX] Clean up legacy dynamo export code ([#149745](https://github.com/pytorch/pytorch/pull/149745))
-- [ONNX] Update default opset to 18 ([#156023](https://github.com/pytorch/pytorch/pull/156023))
+
+#### Default opset in `torch.onnx.export` is now 18 ([#156023](https://github.com/pytorch/pytorch/pull/156023))
+
+When `dynamo=False`, the default ONNX opset version has been updated from 17 to 18. Users can set `opset_version` to explicitly select an opset version.
+
+Version 2.7
+
+```py
+# opset_version=17
+torch.onnx.export(...)
+```
+
+Version 2.8
+
+```py
+# To preserve the original behavior
+torch.onnx.export(..., opset_version=17)
+
+# New: opset_version=18
+torch.onnx.export(...)
+```
+
+#### The `JitTraceConvertStrategy` has been removed ([#152556](https://github.com/pytorch/pytorch/pull/152556))
+
+Support for JIT traced and scripted modules in the ONNX exporter when `dynamo=True` has been removed. You are encouraged to export an nn.Module directly, or create an `ExportedProgram` using `torch.export` before exporting to ONNX.
+
+#### `onnxscript>=0.3.1` is required for the `dynamo=True` option ([#157017](https://github.com/pytorch/pytorch/pull/157017))
+
+You must upgrade `onnxscript` to version 0.3.1 or higher for it to be compatible with PyTorch 2.8.
```

Comment:
Should we specify the reason?

Reply:
I think it's fine? We have a new api in this version that pytorch uses